### PR TITLE
CAPA: Release v25.1.3.

### PR DESCRIPTION
**Pull request by AndiDog**
Towards https://github.com/giantswarm/giantswarm/issues/31843

### Checklist

- [x] Roadmap issue created
- [x] Release uses latest stable Flatcar
- [x] Release uses latest Kubernetes patch version

### Triggering E2E tests

To trigger the E2E test for each new Release added in this PR, add a comment with the following:

`/run releases-test-suites`

If you want to trigger conformance tests, you can do so by adding a comment similar to the following:

`/run conformance-tests PROVIDER=capa RELEASE_VERSION=29.1.0`

For more details see the [README.md](/README.md#running-tests-against-prs).

### CODE REVIEW

**Comment by AndiDog** [comment]
@giantswarm/team-tenet do you want to bump cert-exporter?

File: capa/v25.1.3/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# :zap: Giant Swarm Release v25.1.3 for CAPA :zap:
+
+This release introduced aws-node-termination-handler for graceful draining of nodes during an upgrade or other type of replacement of worker nodes.
```
Comment (Gacko) [inline capa/v25.1.3/README.md:3]:
```suggestion
This release introduces `aws-node-termination-handler` for graceful draining of nodes during an upgrade or other type of replacement of worker nodes.
```

File: capa/v25.1.3/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+The behavior is now explained in the [node pools documentation](https://docs.giantswarm.io/tutorials/fleet-management/cluster-management/node-pools/#what-happens-when-rolling-nodes).
```
Comment (Gacko) [inline capa/v25.1.3/README.md:5]:
```suggestion
Details can be found in the [node pools documentation](https://docs.giantswarm.io/tutorials/fleet-management/cluster-management/node-pools/#what-happens-when-rolling-nodes).
```

File: capa/v25.1.3/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+## Changes compared to v25.1.1
```
Comment (Gacko) [inline capa/v25.1.3/README.md:7]:
```suggestion
## Changes compared to v25.1.2
```

File: capa/v25.1.3/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+### Components
+
+- cluster-aws from v1.1.3 to v1.1.4
+- New application aws-nth-bundle ("NTH" is short for aws-node-termination-handler)
```
Comment (Gacko) [inline capa/v25.1.3/README.md:12]:
```suggestion
- New application `aws-nth-bundle` ("NTH" is short for `aws-node-termination-handler`)
```

Also I'd list this under apps.

File: capa/v25.1.3/release.yaml [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+    version: 2.30.1
+    dependsOn:
+    - cloud-provider-aws
+  - name: aws-ebs-csi-driver-servicemonitors
+    version: 0.1.0
+    dependsOn:
+    - cert-manager
+  - catalog: default
+    name: aws-nth-bundle
+    version: 1.1.1
```
Comment (Gacko) [inline capa/v25.1.3/release.yaml:17]:
```suggestion
  - name: aws-nth-bundle
    version: 1.1.1
```

`default` is already the default via CRD. Also: Are there any dependencies?

**Review by Gacko — CHANGES_REQUESTED** [review]

**Comment by Gacko** [comment]
> @giantswarm/team-tenet do you want to bump cert-exporter?

Yep, will do so.

**Comment by Gacko** [comment]
/run releases-test-suites TARGET_SUITES=./providers/capa/standard

I doesn't really make sense to test the upgrade here as this is the first version after Vintage, right?

**Review by Gacko — APPROVED** [review]

**Comment by Gacko** [comment]
/run releases-test-suites TARGET_SUITES=./providers/capa/standard TARGET_RELEASE=aws-25.1.3

**Comment by Gacko** [comment]
/run releases-test-suites TARGET_SUITES=./providers/capa/standard TARGET_RELEASES=aws-25.1.3

**Comment by AndiDog** [comment]
I still need to fix the aws-node-termination-handler behavior so it reacts to dying spot instances

**Comment by AndiDog** [comment]
/run releases-test-suites TARGET_SUITES=./providers/capa/standard

**Comment by AndiDog** [comment]
/run releases-test-suites TARGET_SUITES=./providers/capa/standard

**Comment by AndiDog** [comment]
Forgot to set the base release for the test

/run releases-test-suites PREVIOUS_RELEASE=25.1.2 TARGET_SUITES=./providers/capa/upgrade

**Review by Gacko — CHANGES_REQUESTED** [review]

**Review by T-Kukawka — APPROVED** [review]

**Review by Gacko — APPROVED** [review]

**Pull request merged**